server/lostandfound/urls.py

```diff
@@ -19,4 +19,5 @@
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
     url(r'^admin', admin.site.urls),
+    url(r'^admin/', admin.site.urls),
```

Comment:
So i guess i will still have to inject back the second route for the other admin url, or what do you feel.